### DISCUSSION TITLE
ı am not accessing my github account

### DISCUSSION THREAD

**Discussion by EdwardKMejia** [community/community#52186] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hello. My account is protected by two factor authentication. I turned on OTP login by Microsoft Authenticator But I reset my phone and lost it. I can't login to my account because there is no otp and recovery code at the moment, what should I do to you?

**Comment by airtower-luna** [comment, marked as answer]
If you have neither TOTP nor your recovery codes, you can't access the account any more. The documentation [is very clear on that](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/recovering-your-account-if-you-lose-your-2fa-credentials). You *can* detach the email address to use it for a new account.

But you wrote "at the moment", if you have the recovery codes but not at hand, best wait until you can get them again.

**Comment by seenakhan** [comment]
Hello,
Recently I also faced the same issue, by  resetting my phone I cant access my github account. There is a solution which I have used to get my github account back. The solution is when we configure the two factor authentication a file named github-recovery-codes.txt will be generated and saved in our computer, open that file and use the latest access code to open your github account. Please let me know this solution works.
Thanks & regards
Seena Khan
